jobs/base_model_validator_errors.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
```

Comment:
Please add simple unit tests that check the contents of the string messages (should be the only time we do so).